src/ViewInstance.js

```diff
@@ -741,17 +738,8 @@ class ViewInstance extends React.Component {
       acc10: areAllFieldsEmpty(values(instanceRelationship)),
     };
 
-    const formattedStatusUpdatedDate = instanceData.instanceStatusUpdatedDate !== '-'
-      ? (
-        <Fragment>
-          <p>
-            <FormattedDate value={instanceData.instanceStatusUpdatedDate} />
-          </p>
-          <p>
-            <FormattedTime value={instanceData.instanceStatusUpdatedDate} />
-          </p>
-        </Fragment>
-      )
+    const formattedStatusUpdatedDate = instanceData?.instanceStatusUpdatedDate !== '-'
+      ? getDateWithTime(instanceData.instanceStatusUpdatedDate)
       : noValue;
```

Comment:
I am confused by the variety of `'-'` values. It is present here in its pure form, in the `noValue` value, and also the `getDateWithTime` function can return it.
What do you think if we make changes to your line 625 and here:
`const formattedStatusUpdatedDate = getDateWithTime(instanceData.instanceStatusUpdatedDate)` ?

Reply:
Agreed. Thanks for your careful attention to detail!